pkg/network/admitter/netsource_test.go

```diff
@@ -93,4 +93,67 @@ var _ = Describe("Validate network source", func() {
 		Expect(causes).To(HaveLen(1))
 		Expect(causes[0].Message).To(Equal("CNI delegating plugin must have a networkName"))
 	})
+
+	It("should reject multiple multus networks with a multus default", func() {
+		spec := &v1.VirtualMachineInstanceSpec{}
+		spec.Domain.Devices.Interfaces = []v1.Interface{
+			*v1.DefaultBridgeNetworkInterface(),
+			*v1.DefaultBridgeNetworkInterface(),
+		}
+		const net1Name = "multus1"
+		const net2Name = "multus2"
+		spec.Domain.Devices.Interfaces[0].Name = net1Name
+		spec.Domain.Devices.Interfaces[1].Name = net2Name
```

Comment:
nit: you could save such iface definition by setting bridge iface with custom name using `libvmi`.

Reply:
It can be improved, but for now it does not focus on a `vmi`, but on a part of its spec.
I guess it is valid to convert this to a `libvmi`now that it is available under pkg.

But it would probably be harder to review if I change it here.
I will consider it as a follow up.